repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/examples/container.cfg.xml

```diff
@@ -32,4 +32,10 @@
 -->
         </ssl-configuration>
     </deployment-config>
+
+    <cluster-config cluster-id="internal-cluster">
```

Comment:
Nothing. It just wouldn't apply the patch. If we wanted to add a check, it would have to be in code... probably an explicit check in the configuration updated method of this service. I expect it would just log the issue. Do we want to do that?